### DISCUSSION TITLE
编译固件没有“无线”选项

### DISCUSSION THREAD

**Discussion by smallprogram** [coolsnowwolf/lede#11549] [Q&A / 快速问答]
无线网卡 4965AGNM，驱动也加了。但是不知道为啥编译出的固件，没有无线选项，网上说要添加luci-app-mtwifi，编译选项里也没有这个，有哪位大佬知道怎么搞吗？
![e777dfad08db09751d5f84d485c538f](https://github.com/coolsnowwolf/lede/assets/6050360/833272ff-4d8d-481c-9baa-6bd301316947)

**Comment by aiamadeus** [comment]


**Reply by smallprogram** [reply]
我试试

**Reply by smallprogram** [reply]
> mtwifi，，，又是谁在瞎扯，你要选 iwlwifi 和网卡对应的 firmware 包。

试了试，依然没有无线......
